src/reducers/video-listing-reducer.js

```diff
@@ -0,0 +1,15 @@
+const videoListReducer = (state, action) => {
+    switch(action.type){
```

Comment:
we can also maintain states for loading & error here. it'll be easier to implement loader on initial rendering.

Reply:
I actually wanted to finish up the basic requirements first, which is why those cases are missing.